Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -4490,6 +4490,22 @@ static bool isEmpty(PDFSelection *selection)
     return !shouldUseInProcessBackingStore() && (layer == layerForHorizontalScrollbar() || layer == layerForVerticalScrollbar() || layer == layerForScrollCorner());
 }
 
+ViewportConfiguration::Parameters UnifiedPDFPlugin::viewportParameters()
+{
+    ViewportConfiguration::Parameters parameters;
+    parameters.width = ViewportArguments::ValueDeviceWidth;
+    parameters.widthIsSet = true;
+    parameters.allowsUserScaling = true;
+    parameters.allowsShrinkToFit = false;
+    parameters.minimumScale = 1;
```

Comment:
Lest we ever change this?
```suggestion
    parameters.minimumScale = minimumZoomScale;
```

Reply:
isn't it 0.2? I think that would be a significant behavior change, which I did not want to make in this patch

Reply:
oh, that's fair

Reply:
Actually, should the minimum scale just be 1 for iOS family?

Reply:
```cpp
#if PLATFORM(IOS_FAMILY)
static constexpr double minimumZoomScale = 1;
#else
static constexpr double minimumZoomScale = 0.2;
#endif
```

Reply:
possible!

Reply:
ok, making this change too yes